package.json

```diff
@@ -6,30 +6,41 @@
   "repository": "alex-e-leon/print-tree",
   "author": "alex leon <alex@leon.st>",
   "license": "MIT",
+  "files": [
+    "lib"
+  ],
   "scripts": {
     "build": "babel index.js -d lib/",
-    "prepublish": "yarn run build",
-    "lint": "./node_modules/.bin/eslint index.js",
-    "test:unit": "mocha --compilers js:babel-core/register test",
-    "test": "yarn run flow && yarn run lint && yarn run test:unit"
+    "prepare": "yarn build",
```

Comment:
use `prepare` as `prepublish` is deprecated in `npm@5`

Reply:
👍 👍  I've been meaning to read up on that